source/source.json

```
@@ -5041,16 +5041,22 @@
 	},
 	{
 		"github": "angband/angband",
-		"title": "A free, single-player roguelike dungeon exploration game",
+		"title": "Angband",
+		"author": "timschumi",
 		"source": "https://github.com/angband/angband",
 		"systems": ["3DS"],
```

Should change this to have DS as well if it supports the DS:

```suggestion
		"systems": ["3DS", "DS"],
```

---

Done